test/support/channel_case.ex

```diff
@@ -18,7 +18,7 @@ defmodule RetWeb.ChannelCase do
   using do
     quote do
       # Import conveniences for testing with channels
-      use Phoenix.ChannelTest
+      import Phoenix.ChannelTest
```

Comment:
The upgrade guide says to replace `use Phoenix.ConnTest` with 
```
import Plug.Conn
import Phoenix.ConnTest
```

Do we need to import Conn as well?

Reply:
I don't think so -- the note for `ChannelTest` only suggests replacing `use` with `import`:

> [ChannelTest] use Phoenix.ChannelTest is deprecated in favor of import Phoenix.ChannelTest
> [ConnTest] use Phoenix.ConnTest is deprecated in favor of import Plug.Conn; import Phoenix.ConnTest
    

  [source](https://github.com/phoenixframework/phoenix/blob/v1.5/CHANGELOG.md#deprecations)